spring-cloud-kubernetes-istio/src/main/java/org/springframework/cloud/kubernetes/istio/IstioAutoConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.cloud.kubernetes.istio;
+
+import io.fabric8.kubernetes.client.Config;
+import me.snowdrop.istio.client.DefaultIstioClient;
+import me.snowdrop.istio.client.IstioClient;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(value = "spring.cloud.istio.enabled", matchIfMissing = true)
```

Comment:
should we specify that the value needs to be `true`?

Reply:
No need. It works without it in the simple case.